packages/nodes-base/credentials/PaddleApi.credentials.ts

```diff
@@ -20,5 +20,11 @@ export class PaddleApi implements ICredentialType {
 			type: 'string' as NodePropertyTypes,
 			default: '',
 		},
+		{
+			displayName: 'Enable Sandbox',
```

Comment:
I suggest changing from `Enable Sandbox` to `Use Sandbox environment API` to make it clear that we're changing the API being used.

Reply:
Yes, that's more significant, Done.